apps/create-fuels-counter-guide/src/components/Info.tsx

```diff
@@ -0,0 +1,47 @@
+import { providerUrl, playgroundUrl } from "../lib.tsx";
+
+export default function Info() {
+  return (
+    <div id="text" className="col-span-3 px-4 py-4">
+      <h1 className="pb-1 pt-0 md:pt-6 text-3xl font-medium">Welcome to Fuel</h1>
+      <p>
+        This Vite + React template was bootstrapped with the{" "}
+        <a
+          href="https://docs.fuel.network/docs/fuels-ts/fuels-cli/#getting-started"
+          target="_blank"
+          className="text-green-500/80 transition-colors hover:text-green-500"
+          rel="noreferrer"
+        >
+          Fuels CLI
+        </a>
+      </p>
+      <p className="pt-8">
+        You are currently connected to:{" "}
+        <a
+          href={providerUrl}
+          target="_blank"
```

Comment:
Should we change this to display the provider URL from which the wallet app is connected? I was connected to testnet but for me, it was showing

![image](https://github.com/user-attachments/assets/902c463c-44cb-451b-b679-bcdf3b1740b2)

Reply:
Not sure about this one. As of now, it shows the URL based on the `VITE_DAPP_ENVIRONMENT` env variable

Reply:
I think the wrong network message will suffice here. That provider is what the contract will be connected to, therefore if a wallet is connected to a different provider it will not be in sync.

Dhai, this is one reason why I thought maybe only allowing a burner wallet locally was a way to go 🤔

Reply:
Resolving as we've now added the wrong network message. It also makes sense to have the provider not driven by the wallet, as it syncs up with the fuels config file.